doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -302,6 +302,10 @@ Matter Bridge
 nRF5340 Audio
 -------------
 
+* Added:
+
+  * Will now use the NFC.TAGHEADER0 value from FICR as the broadcast ID instead of using a random ID.
```

Comment:
```suggestion
  * The application will now use the ``NFC.TAGHEADER0`` value from FICR as the broadcast ID instead of using a random ID.
```
The entry could be moved to the updated section.